nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/test/groovy/org/apache/nifi/web/security/jwt/JwtAuthenticationFilterTest.groovy

```diff
@@ -83,7 +83,7 @@ class JwtAuthenticationFilterTest extends GroovyTestCase {
         String authenticationHeader = "Bearer " + jwtString
 
         // Act
-        boolean isValidHeader = new JwtAuthenticationFilter().validJwtFormat(authenticationHeader)
+        boolean isValidHeader = new NiFiBearerTokenResolver().validAuthorizationHeaderFormat(authenticationHeader)
```

Comment:
The references to `NiFiBearerTokenResolver` should be moved out of this test class.  Recommend that a new test class focus testing on the interface method and not the other methods.